src/pages/Tailwind/Bridge/index.tsx

```diff
@@ -3,7 +3,33 @@ import PageWrapper from 'components/Tailwind/Layout/PageWrapper'
 import PageHeaderLeft from 'components/Tailwind/Layout/PageHeaderLeft'
 import BridgePanel from './BridgePanel'
 import InfoCard from 'components/Tailwind/Cards/InfoCard'
+import PageNotSupported from 'components/Tailwind/Layout/PageNotSupported'
+import { NETWORK_SUPPORTED_FEATURES, NETWORK_LABEL } from '../../../constants/networks'
+import { ChainId } from '@sushiswap/sdk'
+import { useActiveWeb3React } from 'hooks'
 
+const NotSupportedContent = () => {
+  const { chainId } = useActiveWeb3React()
+  const caption = 'Ooops! Sorry this page/feature is not supported in ' + NETWORK_LABEL[chainId as ChainId] + '.'
+  return (
+    <div className="flex items-center bg-white py-6 px-8 border border-primary-hover shadow-md rounded-card">
+      <div className="w-full">
+        <PageNotSupported caption={caption} />
+      </div>
+    </div>
+  )
+}
+const CurrentPanelContent = () => {
+  const { chainId } = useActiveWeb3React()
+  const features = NETWORK_SUPPORTED_FEATURES[chainId as ChainId]
+  let content = <></>
+  if (features?.bridge) {
+    content = <BridgePanel />
+  } else {
+    content = <NotSupportedContent />
+  }
+  return content
```

Comment:
Not 100% sure with react, but we could probably reduce complexity here...

```suggestion
  if (features?.bridge) {
    return <BridgePanel />
  }
  return <NotSupportedContent />
```